setup.py

```diff
@@ -887,16 +888,24 @@ def configure_extension_build():
 
     # Cross-compile for M1
     if IS_DARWIN:
+        report('-- detected IS_DARWIN')
         macos_target_arch = os.getenv('CMAKE_OSX_ARCHITECTURES', '')
+
         if macos_target_arch in ['arm64', 'x86_64']:
+            report('-- detected macos target architecture ' + macos_target_arch)
```

Comment:
```suggestion
            report(f"-- Compiling for  macOS target architecture {macos_target_arch}")
```

Reply:
I changed to `building` as there is `linking` as well as `compiling` and `MacOS` to `Darwin`.